src/paulscode/android/mupen64plusae/input/map/TouchMap.java

```diff
@@ -265,6 +271,28 @@ public int getButtonPress( int xLocation, int yLocation )
     }
     
     /**
+     * Gets the frame for the N64 button with a given asset name
+     * 
+     * @param assetName The asset name for the button
+     * 
+     * @return The frame for the N64 button with the given asset name
+     * 
+     */
+    public Rect getButtonFrame( String assetName )
```

Comment:
In order to support drag and drop I needed a way to get the draw frames for the buttons and analog stick, and *that* needed a way to match the button layout index to the N64 index. Not sure if there's a better way?